network.js

```diff
@@ -169,6 +172,9 @@ module.exports = {
   registerObserver(fn) {
     observers.push(fn);
   },
+  registerProxyUpdatedObserver(fn) {
+    proxyObservers.push(fn);
```

Comment:
@tejohnso is there a scenario where we'd need to call `fn(fields)` like we do in our UI?